src/main/java/me/botsko/prism/Prism.java

```diff
@@ -136,6 +144,12 @@ protected Prism(JavaPluginLoader loader, PluginDescriptionFile description, File
         super(loader, description, dataFolder, file);
     }
 
+    public static void setDebug(boolean debug) {
+        Prism.debug = debug;
+    }
+    public static boolean isDebug() {
```

Comment:
Codacy found an issue: ['METHOD_DEF' should be separated from previous statement.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5911179)